src/xdocs/config.xml

```diff
@@ -729,6 +729,33 @@
       </p>
     </section>
 
+    <section name="Id">
+      <p>
+        Each module has a <code>id</code> property that could be used to differentiate
```

Comment:
`be used to` => `can rename the module name to be a name defined by the user. This is used to`